tests/installation/addon_products_yast2.pm

```diff
@@ -112,11 +116,19 @@ sub run {
                 send_key 'alt-o';                                                   # ok continue
             }
         }
+        if ($perform_reboot eq '1') {
```

Comment:
I guess we can simply check if $perform_reboot is set, and not compare against 1.

Reply:
@rwx788, fixed &, tested here http://dhcp62.suse.cz/tests/2218#

Reply:
That's not exactly what I meant, I suggest using if($perform_reboot). Using defined will return true as you initialize value with 0. I also don't think we need verification run for these small changes.

Reply:
But I removed the initialization with 0, now it uses just "my $perform_reboot;" so imo it should work in same way

Reply:
Oh, sorry, missed that. Yeah, it will work same way.